### DISCUSSION TITLE
如何配置才能不使用{}解构，而是直接导入

### DISCUSSION THREAD

**Discussion by truthtenfold** [unplugin/unplugin-vue-components#878] [General]
### Describe the bug

由于组件库没有按需导出所有组件，我想从它的目录按需导入，如何配置才能实现，例如以下写法
import SelfButton from '@self/component/button'
import SelfForm from '@self/component/form'

### Reproduction

无

### System Info

```Shell
version: 0.21.2
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by kuolemax** [comment]
不知道我的方案适不适用于你的情况，
例如我是用 `vue-echarts` 这个库，
需要 `import VChart from vue-echarts`
我配置成这样可以正确使用
```ts
Components({
  dts: true,
  resolvers: [
    (componentName: string) => {
      if (componentName === 'VChart')
        return 'vue-echarts'
    },
  ],
}),
```
